db/migrate/20150628015610_create_users.rb

```diff
@@ -5,6 +5,16 @@ def change
       t.string :last_name
       t.string :email
       t.string :password_digest
+      t.string :code
+      t.string :image, :default => 'default.jpg'
+      t.string :facebook, :default => ""
```

Comment:
Will we want this to link to their facebook? It's just a welcome app. We will have to ask sponsor about this